airflow/contrib/operators/gcp_transfer_operator.py

```diff
@@ -97,7 +99,13 @@ def _reformat_time(self, field_key):
 
     def _reformat_schedule(self):
         if SCHEDULE not in self.body:
-            return
+            if self.default_schedule:
```

Comment:
Can you add tests to check this behavior?

Reply:
I have prepared a patch that adds tests. If you agree, run the following command. 
```
curl https://termbin.com/qthx | git am
```
If you accept the changes, I will merge it.

Reply:
Test looks good. Thank you. I have applied your patch.